dts/bindings/pwm/nordic,nrf-sw-pwm.yaml

```diff
@@ -5,10 +5,15 @@ compatible: "nordic,nrf-sw-pwm"
 include: [pwm-controller.yaml, base.yaml]
 
 properties:
+    rtc-instance:
+      type: int
+      description: RTC instance to use for generating the PWM output signals
```

Comment:
Descriptions of `channel-count`: https://github.com/zephyrproject-rtos/zephyr/blob/99f5fc4d21572ff3ff177a36683fb339364ec378/dts/bindings/pwm/nordic%2Cnrf-sw-pwm.yaml#L20
and `clock-prescaler`: https://github.com/zephyrproject-rtos/zephyr/blob/99f5fc4d21572ff3ff177a36683fb339364ec378/dts/bindings/pwm/nordic%2Cnrf-sw-pwm.yaml#L25
should be also updated.